disco_aws_automation/disco_autoscale.py

```diff
@@ -165,21 +169,21 @@ def delete_groups(self, hostclass=None, group_name=None, force=False):
                             group.name, exc.message, force)
 
     def clean_groups(self, force=False):
-        '''
+        """
```

Comment:
We can probably just remove this and have people call `delete_groups` directly instead